README.md

```diff
@@ -140,56 +193,50 @@ By default, Sheer will render pages at their natural paths in the project's file
 structure.
 For example, going to <http://localhost:7000/the-bureau/index.html>
 (or <http://localhost:7000/the-bureau/>) renders `/the-bureau/index.html`
-as processed by the [Jinja2](http://jinja.pocoo.org/docs/) templating engine.
-Note that this page does not automatically show any content indexed by Sheer;
+as processed by the [Jinja2](http://jinja.pocoo.org/docs) templating engine.
+**NOTE:** This page does not automatically show any content indexed by Sheer;
 it simply outputs the static HTML written into the template.
 
 ### Outputting indexed content in a Sheer template
 
-Most of our content is indexed from the API output of our WordPress back end. This happens when the `sheer index` command is run.
-(WordPress was previously used to serve the site front-end,
-but going forward, it will only be used as a content editing and storage system.)
-
-If your content isn't being indexed yet, see "Setting up a new WordPress post
-type and processing it with Sheer" on the flapjack/Getting-started-with-Flapjack
-wiki (on our GitHub Enterprise server).
+Most of our content is indexed from the API output of our WordPress back-end. This happens when the `sheer index` command is run.
```

Comment:
@anselmbradford, @Scotchester is a fan of breaking to new lines in an effort to avoid horizontal scrolling in the GitHub diffs and I tend to agree. @Scotchester, what's the character limit you use? 120?